app/views/case_court_reports/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1>Report Categories</h1>
+
+<div class="card card-container">
+  <div class="card-body">
+    <h5 class="card-title"><strong>Generate Court Report</strong></h5>
+    <p class="card-text">
+      The Court Report is pre-filled with information of your case. You can select among currently active cases assigned to you. The document is in Microsoft Word format (.dox).
```

Comment:
Is `.dox` correct? I would expect `.doc` or `.docx` but I might be out of touch with windows

"information of your case" sounds funny - maybe "information *for* your case" instead?

Reply:
@compwron You're right. It's a typo. The correct type is `.docx`.
I'll also change the wordings to "information for your case".